Bottom: be62b0999483abb260160ece559b049b5ebb8a24
Top:    51416b84afe5571fee746a2f5211a066d66ff812
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-03-21 11:51:06 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index c402568..14590b5 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2460,7 +2460,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
